### PR TITLE
feature/1922/hours_column_contain_expected_no._of_days

### PR DESCRIPTION
**Pull request by saurabhpatel123456**
Targets #1922 

### Description
Hours column contains day column
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/EffortTracking/Resources/views/show.blade.php [hunk 1]
```diff
@@ -72,7 +72,10 @@ class="text-theme-body text-decoration-none mb-2 mb-xl-4 align-items-center">
                         <tr>
                             <th scope="col" class="pb-lg-4">Name</th>
                             <th scope="col" class="pb-lg-4">Hours Booked</th>
-                            <th scope="col" class="pb-lg-4">Expected Hours</th>
+                            <th scope="col" class="pb-lg-4">Expected Hours
+                                <div class="ml-lg-3 ml-xl-5 fz-md-10 fz-xl-14"> 
+                                    (1 Days)
```
Comment (P4NK4J) [inline Modules/EffortTracking/Resources/views/show.blade.php:77]: Why we have hardcoded this data ?

**Review by P4NK4J — COMMENTED** [review]
Left a comment, please check.

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 1 above)
Comment (saurabhpatel123456) [reply Modules/EffortTracking/Resources/views/show.blade.php:77]: for insertion of day column into hours column, and also for changes in fte page as well as final page by providing  user input.

File: Modules/EffortTracking/Resources/views/show.blade.php [hunk 2]
```diff
@@ -70,9 +70,13 @@ class="text-theme-body text-decoration-none mb-2 mb-xl-4 align-items-center">
                 <table class="table">
                     <thead>
                         <tr>
+                            {{-- @dd($project) --}}
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Resources/views/show.blade.php:73]: Remove this commented line

File: Modules/EffortTracking/Services/EffortTrackingService.php [hunk 3]
```diff
@@ -26,7 +26,8 @@ public function show($project)
         $workingDays = $this->getWorkingDays($project->client->month_start_date, $currentDate);
         $startDate = $project->client->month_start_date;
         $endDate = $project->client->month_end_date;
-        $totalWorkingDays = count($this->getWorkingDays($startDate, $endDate));
+        $totalWorkingDays = count($this->getworkingDays($startDate, $endDate));
+        $daysTillToday = count($project->getWorkingDaysList($project->client->month_start_date, $currentDate));
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Services/EffortTrackingService.php:30]: How come these 2 functions `getWorkingDaysList`, `getworkingDays` doing the same thing? Do we have duplicate functions?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
PLease check the comments.

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 2 above)
Comment (saurabhpatel123456) [reply Modules/EffortTracking/Resources/views/show.blade.php:73]: ok, done.

File: Modules/EffortTracking/Services/EffortTrackingService.php (hunk 3 above)
Comment (saurabhpatel123456) [reply Modules/EffortTracking/Services/EffortTrackingService.php:30]: no, both function performs different things. the getworkingdayslist used to call daystilltoday function.

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 2 above)
Comment (GauravGusain98) [reply Modules/EffortTracking/Resources/views/show.blade.php:73]: You have not committed this I guess. It is still showing in the chnages

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
PLease check the comment

File: Modules/EffortTracking/Services/EffortTrackingService.php [hunk 4]
```diff
@@ -249,7 +250,7 @@ public function getEffortForProject($project)
                         'project' => $project,
                         'billing_start_date' => $billingStartDate,
                         'billing_end_date' => $billingEndDate,
-                        'sheet_index_for_billable_effort' => $sheetIndexForTotalBillableEffort,
+                     'sheet_index_for_billable_effort' => $sheetIndexForTotalBillableEffort,
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Services/EffortTrackingService.php:253]: Fix the formatting

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor feedbacks. Else looks good. Will approve once the changes are fixed

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**